.github/workflows/_build_test_upload.yml

```diff
@@ -432,6 +432,9 @@ jobs:
           elif [[ $TARGET_FOLDER == tags/* ]]; then
             TARGET_FOLDER=${TARGET_FOLDER:5}
           elif [[ $TARGET_FOLDER == v* ]] && [[ ${{ inputs.pre_dev_release }} == false ]]; then
+            if [[ $TARGET_FOLDER == v*.*.* ]]; then
+              TARGET_FOLDER=${TARGET_FOLDER%.*}
+            fi
```

Comment:
Remove the trailing `.0` from the tag.
`v0.4.0` -> `v0.4`

Reply:
Question: so we will have a single target folder of `v0.4` instead of two like `v0.4.0` and `v0.4.1`?

Reply:
Yeah. This is because minor release should not contain major feature changes unless significant bugs. So, there is no need to create a separate folder for docs.